suite/tests/client-interface/process-id.c

```diff
@@ -0,0 +1,25 @@
+#include <pthread.h>
```

Comment:
Please include the license + copyright header

Reply:
Done.